csharp/src/Apache.Arrow.Adbc/Apache.Arrow.Adbc.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Version>0.1.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\arrow\csharp\src\Apache.Arrow\Apache.Arrow.csproj" />

```

Comment:
I assume this only works locally. Can we reference a nightly build of Apache.Arrow?

Reply:
uses a submodule to obtain. will investigate gh action suggestion